Vizzuality/heco-invest
# Set up Jest

This PR set ups Jest so that unit tests can be executed via GitHub Actions. By upgrading to Next.js 12.1, most of Jest's configuration is handled by Next.js.

## Testing instructions

- You can run the tests manually: `yarn test` (this PR will also be tested before it's deployed on Vercel)
- You can check the application is still working (home and about page)

## Tracking

[LET-127](https://vizzuality.atlassian.net/browse/LET-127).
